db/repair.cc

```diff
@@ -486,6 +486,7 @@ class Repairer {
         if (empty) {
           empty = false;
           t->meta.smallest.DecodeFrom(key);
+          t->min_sequence = parsed.sequence;
```

Comment:
Thanks! Looks like this bug has been here a few years. I guess we made the assertions stronger recently and missed it due to repair_test not included in our test suite (will fix that now).